CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Change Log
 
+## [5.8.0](https://github.com/plivo/plivo-java/tree/v5.8.0) (2021-12-29)
+**Features - Generic**
+- Default log level set to 'NONE'. 
+- Added ability to set log level.
```

Comment:
Please change this to **Enabled log level to be configurable from customers end**

Reply:
Updated changelog